src/Microsoft.PowerShell.ConsoleGuiTools/ModuleLayout.psd1

```diff
@@ -8,9 +8,9 @@
             'publish/NStack.dll'
         )
 
-        'OutGridView.Models' = @(
-            'publish/OutGridView.Models.dll',
-            'publish/OutGridView.Models.pdb'
+        'Microsoft.PowerShell.OutGridView.Models' = @(
+            'publish/Microsoft.PowerShell.OutGridView.Models.dll',
+            'publish/Microsoft.PowerShell.OutGridView.Models.pdb'
```

Comment:
do you publish the PDB too?

Reply:
yeah... why not? :)

Reply:
It's not the nuget standard

Reply:
For information.
From https://devblogs.microsoft.com/dotnet/producing-packages-with-source-link/
> Source Link is easy to add to your projects and we highly recommend that all projects configure it by default.